src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php

```diff
@@ -784,4 +785,14 @@ private function getAddressFormHandler()
     {
         return $this->get('prestashop.core.form.identifiable_object.handler.manufacturer_address_form_handler');
     }
+
+    private function getAlertInfoMessage()
+    {
+        $message = sprintf(
+            'The display of your brands is %s on your store. Go to Shop Parameters > General to edit settings.',
+            $this->configuration->get('PS_DISPLAY_MANUFACTURERS') ? 'enabled' : 'disabled'
+        );
+
+        return $this->trans($message, 'Admin.Catalog.Notification');
```

Comment:
Same question here. ;-)

Reply:
Because the wording is dynamic, I build it first, inside the $message variable, then I use it inside $this->trans.
(is it what you meant ?)

Reply:
Same thing here, yo need to use an `if` condition and two calls to `$this->trans` with hard coded strings.

Reply:
done !